accord-core/src/main/java/accord/topology/TopologyManager.java

```diff
@@ -754,9 +845,21 @@ public long epoch()
         return current().epoch;
     }
 
+    public long firstNonEmpty()
```

Comment:
unused - presumably for integration, should we be using it in burn test? Could leave TODO to expand testing.